src/nvim/option_vars.h

```diff
@@ -366,6 +366,8 @@ EXTERN int p_arshape;           ///< 'arabicshape'
 EXTERN int p_icon;              ///< 'icon'
 EXTERN char *p_iconstring;      ///< 'iconstring'
 EXTERN int p_ic;                ///< 'ignorecase'
+EXTERN char *p_imgprg;          ///< 'imgprg'
+EXTERN char *p_imgprovider;     ///< 'imgprovider'
```

Comment:
drop these for now. push the drop as a new commit, then we will have that in the repo history (won't squash-merge)